sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -413,10 +413,18 @@ CURLcode CurlSession::Perform(Context const& context)
     if (hostHeader == headers.end())
     {
       Log::Write(Logger::Level::Verbose, LogMsgPrefix + "No Host in request headers. Adding it");
-      this->m_request.SetHeader("Host", this->m_request.GetUrl().GetHost());
+      std::string headerValue = this->m_request.GetUrl().GetHost();
```

Comment:
I think the `headerValue` variable is probably better named `hostName` since that's closer to its value.

Reply:
Sure